frontend/src/data/apolloClientInstance.ts

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+
+const apolloClientInstance = new ApolloClient({
+  uri: 'http://localhost:3000/graphql',
```

Comment:
What's the best way to make this configurable for deployment?

Reply:
I've addressed this with environment variables. See `.env` and the new section in the README.